public/components/event_analytics/explorer/log_explorer.tsx

```diff
@@ -159,78 +85,24 @@ export const LogExplorer = ({
     }
   }, []);
 
-  function getQueryTab({
-    tabTitle,
-    tabId,
-    handlesTabClose,
-  }: {
-    tabTitle: string;
-    tabId: string;
-    handlesTabClose: (TabIdToBeClosed: string) => void;
-  }) {
-    return {
-      id: tabId,
-      name: (
-        <>
-          <EuiText size="s" textAlign="left" color="default">
-            <span className="tab-title">{tabTitle}</span>
-            <EuiIcon
-              type="cross"
-              onClick={(e) => {
-                e.stopPropagation();
-                handlesTabClose(tabId);
-              }}
-              data-test-subj="eventExplorer__tabClose"
-            />
-          </EuiText>
-        </>
-      ),
-      content: (
-        <>
-          <Explorer
-            key={`explorer_${tabId}`}
-            pplService={pplService}
-            dslService={dslService}
-            tabId={tabId}
-            savedObjects={savedObjects}
-            timestampUtils={timestampUtils}
-            setToast={setToast}
-            history={history}
-            notifications={notifications}
-            savedObjectId={savedObjectId}
-            tabCreatedTypes={tabCreatedTypes}
-            curSelectedTabId={curSelectedTabIdRef}
-            http={http}
-            searchBarConfigs={searchBarConfigs}
-            queryManager={queryManager}
-          />
-        </>
-      ),
-    };
-  }
-
-  const memorizedTabs = useMemo(() => {
-    const res = map(tabIds, (tabId) => {
-      return getQueryTab({
-        tabTitle: tabNames[tabId] || TAB_TITLE,
-        tabId,
-        handlesTabClose: handleTabClose,
-      });
-    });
-
-    return res;
-  }, [tabIds, tabNames, tabCreatedTypes]);
-
   return (
     <>
-      <EuiTabbedContent
-        id="queryTabs"
-        className="queryTabs"
-        tabs={memorizedTabs}
-        selectedTab={memorizedTabs.find((tab) => tab.id === curSelectedTabId)}
-        onTabClick={(selectedTab: EuiTabbedContentTab) => handleTabClick(selectedTab)}
-        data-test-subj="eventExplorer__topLevelTabbing"
-        size="s"
+      <Explorer
+        key={`explorer_${tabIds[0]}`}
+        pplService={pplService}
+        dslService={dslService}
+        tabId={tabIds[0]}
+        savedObjects={savedObjects}
+        timestampUtils={timestampUtils}
+        setToast={setToast}
+        history={history}
+        notifications={notifications}
+        savedObjectId={savedObjectId}
+        tabCreatedTypes={tabCreatedTypes}
+        curSelectedTabId={curSelectedTabIdRef}
+        http={http}
```

Comment:
Do we need http to be passed here can we use import `CoreRefs`?

Reply:
I just copied the entire component over for tab changes. But to your point, I've created the [issue](https://github.com/opensearch-project/dashboards-observability/issues/975) for tracking this as I saw there are around 7 - 8 references in explorer where changes required for this are kinda out of the scope of this PR. Later we can have campaign for this as I feel we need to also refactor some of the components to not necessarily need http